app.js

```diff
@@ -11,3 +11,171 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
+    featuredImage: '/images/Snapshoot-Portfolio2.png',
+    tech1: 'html',
+    tech2: 'css',
+    tech3: 'javascript',
+    modalId: 'card3',
+    liveVersionLink: 'https://github.com/hadi111jafari',
+    sourceLink: 'https://github.com/hadi111jafari',
+  },
+  {
+    name: 'Uber Navigation',
+    company: 'CANOPY',
+    stack: 'Back End Dev',
+    year: '2015',
+    description: `
+    A smart assistant to make driving more safe, efficient, and fun by
+    unlocking your most expensive computer: your car. `,
+    featuredImage: '/images/Snapshoot-Portfolio2.png',
+    tech1: 'html',
+    tech2: 'css',
+    tech3: 'javascript',
+    modalId: 'card4',
+    liveVersionLink: 'https://github.com/hadi111jafari',
+    sourceLink: 'https://github.com/hadi111jafari',
+  },
+];
+
+function newCard(card) {
+  return `
+    <div class="card">
+        <img src="${card.featuredImage}" alt="" />
+        <div class="card-content">
+          <h2>${card.name}</h2>
+          <div class="list-1">
+            <p>${card.company}</p>
+            <ul class="field">
+              <li class="backend">${card.stack}</li>
+              <li class="year">${card.year}</li>
+            </ul>
+          </div>
+
+          <p>
+          ${card.description}
+          </p>
+          <ul class="languages">
+            <li>${card.tech1}</li>
+            <li>${card.tech2}</li>
+            <li>${card.tech3}</li>
+          </ul>
+          <div id="btn-curser">
+          <button onclick="openModal('${card.modalId}')" class="btn modal-btn" id="${card.modalId}">See Project</button>
+            
+          </div>
+        </div>
+      </div>
+  `;
+}
+function openModal(modalId) {
+  const btnOpen = document.querySelector(`#${modalId}`);
+  const modalOverlay = document.querySelector('.modal-overlay');
+  btnOpen.addEventListener('click', () => {
+    modalOverlay.classList.add('show-container');
+  });
+  cards.forEach((card) => {
+    if (card.modalId === modalId) {
+      const x = document.querySelector('.modal-container');
+      x.innerHTML = `
+  <div class="card">
+  <div class="card-content">
+  <h2>${card.name}</h2>
+  <div class="list-1">
+  <p>${card.company}</p>
+  <ul class="field">
+  <li class="backend">${card.stack}</li>
+  <li class="year">${card.year}</li>
+  </ul>
+  </div>
+  <img src="${card.featuredImage}" alt="" />
+    
+    <p>
+    ${card.description}
+    </p>
+    <ul class="languages">
+      <li>${card.tech1}</li>
+      <li>${card.tech2}</li>
+      <li>${card.tech3}</li>
+    </ul>
+    <div id="btn-curser">
+    <div style="display:flex;">
+    <div class="SLB">
+        <a href="${card.liveVersionLink}">LIve version</a>
+        <i>icon goes here</i>
+    </div>
+    <div>
+        <a href="${card.sourceLink}">source code</a>
+        <i>icon goes here</i>
+    </div>
+</div>      
+    </div>
+  </div>
+</div>
+<button class="btn-close"><i class="fas fa-times"></i></button>
+`;
+    }
+  });
```

Comment:
- [ ] As required in the project requirements, you must stick to the pop-up design as much as possible (margins top, bottom, right, and left, font and font sizes, layouts... ) using the Figma Template provided.
- Your design:
<img width="946" alt="image" src="https://user-images.githubusercontent.com/77190203/157998420-b50b1554-1c82-4b81-b899-57360e8fadc0.png">
- Figma design:
<img width="407" alt="image" src="https://user-images.githubusercontent.com/77190203/157998368-46af831e-62b0-4f27-b844-d9d1c91e8673.png">

- This module is about HTML & CSS, and the Figma template is very detailed. Sticking to the design will help you practice your HTML & CSS skills better.